### PR TITLE
Contains_duplicate file DSA problem added

### LINKED ISSUE #1677: Contains duplicate DSA problem adding.
### Issue Summary

Contains duplicate DSA problem adding. using different different data structure.

### Issue Description

Contains duplicate DSA problem adding. using different different data structure.

### Proposed Solution (Optional)

_No response_

### Priority

High - Requires urgent attention

### Category

- [ ] Enhancement
- [ ] Refactor
- [ ] Security
- [ ] Design
- [ ] Other

### Additional Context (Optional)

_No response_

### PR DESCRIPTION
**Pull request by PradeepFSTdhane123**
Contains_duplicate file DSA problem added

### Description
Contains_duplicate file DSA problem added  with full explanation and approach.

Fixes #1677

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/DSA-Problem-Solution/Contains_duplicate.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+Check if an element exists (lookup) in O(1) average time.
+**2. Iterating Through the Array:**
+- The algorithm iterates through each element of the array.
+For each element:
+It checks if the element already exists in the set (using the find() method). If it does, this means the element is a duplicate, so we return true.
+If the element is not found in the set, it is added to the set to track that we’ve seen it before.
+**3. Returning the Result:**
+- If the loop completes without finding any duplicates, the function returns false, meaning all elements in the array are distinct.
+Code Flow:
+Initialization: We declare an unordered_set<int> uniques to store the unique numbers encountered during the iteration.
```
Comment (ajay-dhangar) [inline docs/DSA-Problem-Solution/Contains_duplicate.md:85]: replace `unordered_set<int>` with ``` `unordered_set<int>` ```

File: docs/DSA-Problem-Solution/Contains_duplicate.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
+---
+id: Contains-Duplicate-leetcode-217
```
Comment (ajay-dhangar) [inline docs/DSA-Problem-Solution/Contains_duplicate.md:2]: We use the `id` value as a unique `url`.  So, we need write it in small letters.

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
Update code based on suggestions

**Comment by PradeepFSTdhane123** [comment]
@ajay-dhangar  changes successfully added please check pr #1802

**Pull request closed without merging**